infrastructure/resource_group_level/modules/trade-finance-manager-api.bicep

```diff
@@ -0,0 +1,247 @@
+  GOV_NOTIFY_EMAIL_RECIPIENT: 'test-value'
+}
+
+
+// https://learn.microsoft.com/en-us/azure/virtual-network/what-is-ip-address-168-63-129-16
+var azureDnsServerIp = '168.63.129.16'
+
+// These values are hardcoded in the CLI scripts
+var settings = {
+  WEBSITE_DNS_SERVER: azureDnsServerIp
+  WEBSITE_VNET_ROUTE_ALL: '1'
+  PORT: '5000'
+  WEBSITES_PORT: '5000'
+}
+
+// These values are taken from an export of Configuration on Dev (& validating with staging).
+var additionalSettings = {
+  // Note that the dev & staging didn't have AI enabled
+  // If enabling, consider using APPLICATIONINSIGHTS_CONNECTION_STRING instead
+  // APPINSIGHTS_INSTRUMENTATIONKEY: applicationInsights.properties.InstrumentationKey
+  // APPLICATIONINSIGHTS_CONNECTION_STRING: applicationInsights.properties.ConnectionString
+  
+  DOCKER_ENABLE_CI: 'true'
+  DOCKER_REGISTRY_SERVER_URL: '${containerRegistryName}.azurecr.io'
+  DOCKER_REGISTRY_SERVER_USERNAME: dockerRegistryServerUsername
+  LOG4J_FORMAT_MSG_NO_LOOKUPS: 'true'
+  TZ: 'Europe/London'
+  WEBSITE_HTTPLOGGING_RETENTION_DAYS: '3'
+  WEBSITES_ENABLE_APP_SERVICE_STORAGE: 'false'
+  
+  // All of these were found on dev but not staging
+  DTFS_CENTRAL_API_URL: 'TODO:FN-424' // https://tfs-dev-dtfs-central-api.azurewebsites.net/
+  // Note that EXTERNAL_API_URL is also in Connection Strings! (via CLI)
+  EXTERNAL_API_URL: 'TODO:FN-424' // https://tfs-dev-external-api.azurewebsites.net/ 
+  PORTAL_API_URL: 'TODO:FN-424' //https://tfs-dev-portal-api.azurewebsites.net/
+  TFM_API_URL: 'TODO:FN-424' // https://tfs-dev-trade-finance-manager-api.azurewebsites.net/
+}
+
+var nodeEnv = environment == 'dev' ? { NODE_ENV: 'development' } : {}
+
+var appSettings = union(settings, secureSettings, additionalSettings, additionalSecureSettings, nodeEnv)
+
+var tfmApiName = 'tfs-${environment}-trade-finance-manager-api'
+var privateEndpointName = 'tfs-${environment}-trade-finance-manager-api'
+var applicationInsightsName = 'tfs-${environment}-trade-finance-manager-api'
+
+
+var connectionStringsList = [for item in items(union(secureConnectionStrings, additionalSecureConnectionStrings)): {
+  name: item.key
+  value: item.value
+} ]
+
+var connectionStringsProperties = toObject(connectionStringsList, item => item.name, item => {
+  type: 'Custom'
+  value: item.value
+} )
+
+
+// Then there are the calculated values. 
+var mongoDbConnectionString = replace(cosmosDbAccount.listConnectionStrings().connectionStrings[0].connectionString, '&replicaSet=globaldb', '')
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' existing = {
+  name: storageAccountName
+}
+var storageAccountKey = storageAccount.listKeys().keys[0].value
+
+// Note that in the CLI script, http was used, but the value in the exported config was https.
+var externalApiUrl = 'https://${externalApiHostname}'
+
+var dtfsCentralApiUrl = 'http://${dtfsCentralApiHostname}'
+
+var connectionStringsCalculated = {  
```

Comment:
Again, I think it is strange to have all these values as connection strings. We can change that after we've taken over the environments though.

Reply:
agreed